components/browser_kit/introduction.rst

```diff
@@ -0,0 +1,227 @@
+.. index::
+   single: BrowserKit
+   single: Components; BrowserKit
+
+The BrowserKit Component
+========================
+
+    The BrowserKit component simulates the behavior of a web browser, allowing
+    you to make requests, click on links and submit forms programmatically.
+
+Installation
+------------
+
+You can install the component in two different ways:
+
+* :doc:`Install it via Composer </components/using_components>`
+  (``symfony/browser-kit`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/BrowserKit).
```

Comment:
after the normalisation done recently the real URL is https://github.com/symfony/browser-kit